### PR TITLE
Replace oTypographySerifItalic in brand

### PR DESCRIPTION
**Pull request by laurieboyes**
Forgivable? `oTypographySerifItalic` is dead as far as Next is concerned, as FinancierDisplayText is no longer an approved font: https://github.com/Financial-Times/next-sass-setup/blob/master/_approved-fonts.scss

Currently we rely on the `serif` font family for fallback, which is overridable (as pointed out by this Jira issue: https://jira.ft.com/browse/NFT-482)

A similar fix has been put in place on homepage: https://github.com/Financial-Times/n-card/blob/fd03d015577075db11056f72df919e52640edb8a/components/_card-asymmetric.scss#L8-L15

Affects this bit:
screen shot 2016-06-09 at 15 10 07

### CODE REVIEW

**Comment by bjfletcher** [comment]
IMHO, if that's been done in homepage then we should do the same. :+1:

File: src/scss/_typography.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@
 }
 
 .n-card__brand {
-	@include oTypographySerifItalic(s);
+	font: italic 13.5px/17px Georgia, serif;
```
Comment (laurieboyes) [inline src/scss/_typography.scss]: Yeah you're right, setting to 13 as discussed

**Comment by keirog** [comment]
👍

**Comment by pauloneillft** [comment]
@laurieboyes a word of warning. Georgia Italic renders woefully on IE. I'd run this by a visual designer

**Comment by laurieboyes** [comment]
Oh man really, Ok I'll have a look cheers

**Pull request merged**